ouroboros-consensus-diffusion/ouroboros-consensus-diffusion.cabal

```diff
@@ -155,6 +155,7 @@ library unstable-mock-testlib
     , cardano-crypto-class
     , containers
     , ouroboros-consensus:{ouroboros-consensus, unstable-consensus-testlib, unstable-mock-block}
+    , ouroboros-network-api
```

Comment:
Do we want this dependency or would we rather have a mock type without the `SizeInBytes` wrapper/reexport the internals somewhere in `ouroboros-consensus`?